docker-compose.yaml

```diff
@@ -40,8 +42,6 @@ services:
 
   ffc-demo-payment-message-queue:
     image: softwaremill/elasticmq
-    ports:
```

Comment:
Why remove the port?  I find it useful for local development to be able to send test messages to ElasticMQ

Reply:
The port was moved, rather than removed. It's been pushed down to the `docker-compose.override.yaml` so will still be available when running 'docker-compose up', but not when running tests, that don't apply the override file.

It's the same approach we use for the ports to access the API or website URL. The problem with having the port exposed in the `docker-compose.yaml` is this gets bound on the build server while tests are running. If two projects attempt to bind the same port one of them will error and the build will fail.